app-admin/haskell-updater/haskell-updater-1.2.10.ebuild

```diff
@@ -8,7 +8,7 @@ inherit eutils haskell-cabal
 
 DESCRIPTION="Rebuild Haskell dependencies in Gentoo"
 HOMEPAGE="https://wiki.haskell.org/Gentoo#haskell-updater"
-SRC_URI="mirror://hackage/packages/archive/${PN}/${PV}/${P}.tar.gz"
+SRC_URI="https://hackage.haskell.org/package/${PN}-${PV}/${P}.tar.gz"
```

Comment:
`${PN}-${PV}/${P}.tar.gz` looks confusing here. I don't see why not `${P}/${P}.tar.gz`

Reply:
I went for the minimal change in my sed, didn't think of actually replacing that. But'll do.